test/unit/gateways/mercado_pago_test.rb

```diff
@@ -329,6 +329,104 @@ def test_includes_issuer_id
     assert_equal '4141491|1.0', response.authorization
   end
 
+  def test_purchase_includes_taxes_array
```

Comment:
Do you think it would be worthwhile to add a test for an array containing multiple objects? That seems like the most complicated use case.

Reply:
One of the failure cases has multiple objects, but I will add that in the successful case as well :+1: